src/pages/ModuleEditor/ModuleEditorSchema.ts

```diff
@@ -0,0 +1,13 @@
+import {object, string} from 'yup';
+
+export interface ModuleEditorForm {
+  moduleName: string;
+  shareLink: string;
+  description: string;
+}
+
+export const ModuleEditorSchema = object<ModuleEditorForm>({
```

Comment:
This Schema should be base don the Module interface. You can find it in types/Module.ts